platforms/kops_aws.env

```diff
@@ -8,7 +8,7 @@ KUBE_SSH_USER=admin
 LOG_DUMP_SSH_KEY=/workspace/.ssh/kube_aws_rsa
 LOG_DUMP_SSH_USER=admin
 LOG_DUMP_SAVE_LOGS=cloud-init-output
-LOG_DUMP_SAVE_SERVICES=protokube
+LOG_DUMP_SAVE_SERVICES="protokube kops-configuration"
```

Comment:
I don't know, and am happy to switch if you're saying "you don't need the quote here" rather than "do we need the quote here?"

I used quotes because the only example I could find used quotes:

In kubernetes: 

```
> git grep LOG_DUMP_SAVE_SERVICES
cluster/log-dump.sh:    local -r services=( ${systemd_services} ${opt_systemd_services} ${LOG_DUMP_SAVE_SERVICES:-} )
```

```
> git grep systemd_services
cluster/log-dump.sh:readonly node_systemd_services="node-problem-detector"
cluster/log-dump.sh:readonly systemd_services="kubelet docker"
...
```